macros/src/auto_args.rs

```diff
@@ -101,6 +147,11 @@ pub trait WrapAsync<B: Send + Sync + 'static> {
 	fn wrap_rpc(&self, base: &B, params: Params) -> BoxFuture<Value, Error>;
 }
 
+/// Wrapper trait for meta RPC functions.
+pub trait WrapMeta<B: Send + Sync + 'static, M: Metadata> {
+	fn wrap_rpc(&self, base: &B, params: Params, meta: M) -> BoxFuture<Value, Error>;
```

Comment:
IMO it should all be async now that futures are integrated. But I suppose this limitation is fine.